src/mutation/__tests__/RelayMutationQuery-test.js

```diff
@@ -43,6 +43,11 @@ describe('RelayMutationQuery', () => {
       node => !node.getSchemaName || node.getSchemaName() !== 'source';
     return filterRelayQuery(RelayTestUtils.getNode(...args), filterCallback);
   }
+  function getMutationWithoutClientId(mutation) {
```

Comment:
If you taught `RelayQLPrinter` not to print `clientMutationId` for mutation queries, could you get rid of all of this?